src/js/tabs/vtx.js

```diff
@@ -960,8 +960,9 @@ TABS.vtx.initialize = function (callback) {
             powersString += "[" + power_list[index].value + "]=" + power_list[index].label + ", ";
         }
         powersString += "}\n";
-        let text = frequencieString + freqBandsString + bandsString + powersString;
-        return text;
+        const deviceTable = "deviceTable = { [1]=\"6705\", [3]=\"SA\", [4]=\"Tramp\", [255]=\"None\" },\n";
```

Comment:
It is not ideal that we are introducing more dependency on configurator than needed into the lua scripts here.
Is it possible to move `deviceTable` and `pitModeTable` into a different file to avoid this? This will maintenance of the lua scripts easier down the track.

Reply:
Yes - this was needed due to the changes in betaflight/betaflight-tx-lua-scripts#284 so it could be fixed there (which is probably the better place for it tbh) I'll leave this open until it's decided one way or another...

Reply:
Maybe we should move ``deviceTable`` and ``pitModeTable`` back into vtx.lua and reserve vtx_defaults/modelname.lua for tables and variables specific to vtx tables?

Reply:
SGTM